float32[7] jntPos
float32[7] jntVel
float32[7] addJntTrq 